docs/src/content/en/docs/workflows-vnext/inngest-workflow.mdx

```diff
@@ -7,59 +7,84 @@ description: "Inngest workflow allows you to run Mastra vNext workflows with Inn
 
 [Inngest](https://www.inngest.com/docs) is a developer platform for building and running background workflows, without managing infrastructure.
 
-## Setup
+## How Inngest Works with Mastra
 
-```sh
-npm install @mastra/inngest @mastra/core @mastra/deployer @hono/node-server
-```
+Inngest provides an event delivery and orchestration layer for running Mastra vNext workflows in response to events. The integration connects Inngest’s event system to Mastra’s workflow engine, enabling event-driven execution, step-level monitoring, and suspend/resume functionality.
 
-### Local Development Environment
+### Integration Architecture
 
-Inngest offers two methods for local development:
+- **Workflow Registration:**  
+  Workflows are defined using the `createWorkflow` and `createStep` helpers from `@mastra/inngest`. These helpers produce workflow and step objects compatible with both Mastra and Inngest.
 
-#### Option 1: Using Docker
+- **Event Handling:**  
+  Each workflow is registered as an Inngest function. When an event matching the workflow’s trigger is received, Inngest invokes the workflow handler. The handler starts or resumes a workflow run using the event payload as input.
 
-Run Inngest locally using Docker on port 8288, configured to listen for events on port 3000:
+- **Run Management:**  
+  Workflow runs are managed by the `InngestRun` class. This class sends events to Inngest to start or resume runs, polls for run status, and retrieves results. Workflow state is persisted using Mastra’s storage system.
```

Comment:
Let's make a note that the interface is the same here. This could be combined with ^ as well tbh.